### ISSUE TITLE
mol.intor('int1e_rinv') Integral with single Gaussian BF

### ISSUE THREAD

**Issue by manassharma07** [pyscf/pyscf#505]
Hi,
Is it possible to perform an integral like: `mol.intor('int1e_rinv')`, 
but for just a single Gaussian.
I tried doing this by making the mol have the first atom as the Ghost atom at `R=0,0,0` with a single basis function of 1s Gaussian whose exponent=0 and contraction coefficient= 1. 
Ex:
```
mol = gto.Mole()
     mol.atom = '''Ghost 0 0 0; '''
     mol.basis = {'Ghost': gto.basis.parse('''
     Ghost    S
          0.0              1.0
     ''')}
```
Then, after running `mol.build()`, the normalisation factor becomes 0, which I fix by doing
`mol._libcint_ctr_coeff(0)[:]=1.0`

The remaining atoms of the mol object contain the atoms I need as usual, with normal basis functions. 

After doing, `A = mol.intor('int1e_rinv')`,
I was hoping to use the integrals in the first row (except the first element [0,0]), i.e. `A[0,1:]`

Does this approach make sense? I wasn't able to get the desired results, so I am hoping if someone could help me with this.

**Comment by gkc1000** [comment]
Just take the exponent of the Gaussian you want (rho) and divide it by 2
(-> phi), then set up a molecule with that Gaussian, i.e. int phi^2 1/r =
int rho 1/r, adjusting for normalization. But it's probably easiest to just
use the formula for the nuclear integral (see e.g. Szabo and Ostlund).

**Comment by manassharma07** [comment]
Hi, 
Thanks for the quick reply. I did actually think about doing it the way you mention, however, it requires for me to calculate the Normalization factor myself in order to take care for it.
**For example**: If I create a mol, with the single Gaussian basis: `exp=1, and ctr=1`, then the Normalization factor _N_ is calculated for it accordingly. Now if I change the exponent to 0.5,
`mol.bas_exp(0)[:] = 0.5`  , and then calculate the overlap, the answer of the overlap integral would be enhanced by a factor of _N_(the original normalization factor). So I need to take care of it by dividing the answer by _N_. So, I would need to add some code to calculate the Normalization factors for different _l_ and also, the Normalization factor of the contraction. 
Isn't there a way to get the Normalization factor from Pyscf itself?
From what I understand, gto_norm() only does a portion of the complete Normalization and the remaining stuff is done by libcint. 

This is exactly, why I was having a problem with my initial approach, where I made the coefficient sent to libcint =1: mol._libcint_ctr_coeff(0)[:]=1.0, but didn't get the expected result, as libcint added some factor on its own while calculating the integral.
I figured out the factor (=3.5449...) by comparing the expected result with the answer I got.
So, I have been able to resolve the issue, but still have a question.

It's very much possible that I am doing something wrong. 

Is it really not possible to modify the coefficients(normalization factor*contraction coefficient) ourselves, and only use libcint for the calculation of the integral???

Really appreciate the help so far, and apologies for stupid questions.
